user/contactmethod/store.go

```diff
@@ -267,31 +186,35 @@ func (s *Store) DeleteTx(ctx context.Context, tx *sql.Tx, ids ...string) error {
 	}
 
 	if permission.Admin(ctx) {
-		_, err = wrapTx(ctx, tx, s.delete).ExecContext(ctx, sqlutil.UUIDArray(ids))
+		uids := make([]uuid.UUID, len(ids))
+		for i, id := range ids {
+			uids[i] = uuid.MustParse(id)
+		}
+		err = gadb.New(tx).DeleteContactMethod(ctx, uids)
 		return err
 	}
 
-	rows, err := wrapTx(ctx, tx, s.lookupUserID).QueryContext(ctx, sqlutil.UUIDArray(ids))
+	uids := make([]uuid.UUID, len(ids))
+	for i, id := range ids {
+		uids[i] = uuid.MustParse(id)
```

Comment:
you could use `validate.ParseManyUUID()` here instead of a loop, that would also give an error instead of panicking